packages/design-preview/src/components/sidebar.tsx

```diff
@@ -54,7 +56,7 @@ const Sidebar: React.FC< SidebarProps > = ( {
 	const translate = useTranslate();
 	const [ isShowFullDescription, setIsShowFullDescription ] = useState( false );
 	const isShowDescriptionToggle = shortDescription && description !== shortDescription;
-	const navigatorButtons = useNavigatorButtons( screens );
+	const navigatorButtons = useNavigatorButtons( screens, isExternallyManaged );
```

Comment:
I think it'd be better to change the `useScreens` hook and return an empty array for marketplace themes. That way we won't need to change the `useNavigatorButtons` hook.